plugin/keyring_vault/vault_memory.h

```diff
@@ -7,13 +7,17 @@
 
```

Comment:
winnt.h is included in my_global.h by chain : my_global.h -> winsock2.h ->windows.h ->winnt.h
string.h is always needed for Secure_string definition - it uses std::basic_string
as this stays, there is no need to include <memory> separately.